src/README.md

```diff
@@ -126,6 +126,14 @@ implemented the app instance is discarded and a new instance is created.
 *Optional*. Print a report of the current app status.
 
 
+— Method **myapp:start**
+
+*Optional*. Starts the app.
+
+At this point links and app name are set. The app has a chance to do
+additional initialization if needed.
```

Comment:
Right, only the arrays with the links are initialized (`self.input` and `self.output`) but the arrays are empty. Here is a good chance to do anything related with the app name which is the only thing that is available in `start` and it's not in `:new()`. That's the only thing I can think for now.